Scripts/Editor/Core/ScriptableObjectCollectionSettings.cs

```diff
@@ -21,8 +17,14 @@ public class ScriptableObjectCollectionSettings : ScriptableObjectForPreferences
         private bool useMaximumNamespaceDepth = true;
         public bool UseMaximumNamespaceDepth => useMaximumNamespaceDepth;
 
-        [SerializeField] private int maximumNamespaceDepth = 2;
+        [SerializeField] 
+        private int maximumNamespaceDepth = 2;
         public int MaximumNamespaceDepth => maximumNamespaceDepth;
+        
+        [SerializeField]
+        private string generatedScriptsDefaultFilePath = "Assets\\Generated\\Scripts";
```

Comment:
Worth to consider throwing an @ in front of that bad boy so you can use \ 👀 

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/verbatim